cockatrice/src/remoteclient.cpp

```diff
@@ -365,7 +367,9 @@ void RemoteClient::disconnectFromServer()
 QString RemoteClient::getSrvClientID(const QString _hostname)
 {
     QString srvClientID = settingsCache->getClientID();
-    srvClientID += _hostname;
+    QHostInfo hostInfo = QHostInfo::fromName(_hostname);
+    QHostAddress hostAddress = hostInfo.addresses().first();
```

Comment:
What logic would you prefer if it does fail?  

Currently if it fails we append nothing to the end of the mac address section of the clientid and it gets hashed.  if it doesnt fail (success) we append the ip address found.  Fall back to name possibly?  IMO, if the lookup fails chances are the client will never connect to the cockatrice server they intend on connecting to anyway because Qt is using the underlying operating system functionality to perform the lookups so if the OS is failing to look things up then the app isnt getting to its end point in general.

Or are you thinking like a code handling type check?